onnx_tf/common/pooling_helper.py

```diff
@@ -189,7 +189,15 @@ def _loop_over_output(batch, channel):
                 else:
                     if val > maxval:
                         maxval = val
-                        ind = 0
+                        # batch_offset = batch * C * H * W
+                        batch_offset = batch
+                        for d in input_shape[1:]:
+                          batch_offset *= d
+                        # channel_offset = channel * H * W
+                        channel_offset = channel
+                        for d in input_shape[2:]:
+                          channel_offset *= d
+                        ind = batch_offset + channel_offset
```

Comment:
can we optimize this by calculating the H*W* ... ahead of time and using it during the maxpool operation. something like this:

    def _loop_over_output(batch, channel):
        dims = [range(output_sp_shape[d]) for d in range(spatial_size)]
        **image_size = 1
        for d in input_shape[2:]:
          image_size *= d**
        ............................................................
                else:
                    if val > maxval:
                        maxval = val

                        **ind = image_size * (batch * input_shape[1] + channel)**
                 ...................